src/core_modules/capture-core/components/WidgetNote/WidgetNote.types.ts

```diff
@@ -0,0 +1,24 @@
+export type NoteType = {
+    value: string;
+    storedAt: string;
+    createdBy?: {
+        firstName: string;
+        surname: string;
+    };
+};
+
+export type NoteSectionProps = {
+    notes: Array<NoteType>;
+    handleAddNote: (text: string) => void;
+    placeholder: string;
+    emptyNoteMessage: string;
+    classes?: Record<string, string>;
+};
```

Comment:
Create a new file `NoteSection.types.ts` in the `/NoteSection` folder and move these types definitions